### ISSUE TITLE
Where are Equalify's limits? What are we trying to achieve?

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#164]
I launched Equalify to be the WordPress or Drupal of Accessibility.

My mission is to solve a key problem: accessibility scanning is inaccessible. Institutions must pay large amounts of cash for technological solutions that are very buildable.

With Equalify, anyone with WordPress or Drupal experience can scan their sites for accessibility issues and develop integrations that everyone benefits from.

Now, 305 days after my first commit, I'm trying to nail down an underlining philosophical question: 

_**Where are Equalify's limits? What are we trying to achieve?**_

We can code anything we want. Limits dictate our possibilities.

Here are my observations thus far:

1. **People love that Equalify exists.** Everyday users tell me, "I've been dreaming of a solution like Equalify." When I ask them what that dream was, they answer: "an easy-to-use open-source platform." We have to keep Equalify accessible.
2. **Accessibility starts at the code.** I am always trying to make the code behind Equalify easier to use. Anyone can build an open-source project, but keeping the code understandable sets Equalify apart from other projects.
3. **Passion has limits.** I promised myself I wouldn't think about money in 2022. Every time I make an Equalify commit, I feel I am doing something good. Only recently has that feeling turned to dread. I've begun trying to think about sustainability instead of just passion (see https://github.com/bbertucc/equalify/issues/131). Hopefully, this thought will lead to full-time work on Equalify, but I am trying to be open to taking contracts outside Equalify's scope.

A few sub-questions:
- **Does an easy installation process matter?** Lately, I've been considering integrating Rust-made components that would complicate the installation process. The benefit is faster scan times. Does that matter?
- **What should we be striving for?** I'm still laser-focused on being WordPress or Drupal for accessibility. Does that goal mean anything?
- **What am I missing?** Often, I wake up in the middle of the night with an "AHA!" The more I've met with users, the less my sleep has been disturbed. Users are adding great ideas to Equalify, and I hope to increase that feedback.
- **What's next?** I'm just plodding away. Meeting new users every day and filing bugs as find time to code in between client work. Maybe I should take a different tact?

Thanks to everyone for coming on this adventure with me. I sincerely appreciate the great feedback you've given thus far.

Onwards and upwards!

**Comment by bbertucc** [comment]
Too philosophical!

**Comment by bbertucc** [comment]
Re-opening this issue. Building #131 and answering #176, two core interests pop: 

**CORE INTEREST 1**: We should build **the most useful** accessibility platform.

**CORE INTEREST 2**: We should also build **the most open** accessibility platform.

With usefulness and openness at Equalify's heart, the project will attract top talent who know that proprietary solutions prevent new technologies from making the internet more accessible.

Let me tackle both values:

## Usefulness

Usefulness can be defined by one key question: Are users using the features? That question can be answered by analytics and user testing tied to every Equalify feature.

Perhaps we should add a line to [CONTRIBUTING.md](https://github.com/bbertucc/equalify/blob/main/CONTRIBUTING.md) that says:
> Equalify aims to release the most useful features for accessibility testing and remediation. We do this by validating feature ideas in user tests. The user tests must further the conviction that the feature is useful.

That guideline could lead to standards around what defines a "useful feature"

## Openness

One key way Equalify that demonstrate our commitment to openness is by only supporting open-source solutions. Here is a sample [CONTRIBUTING.md](https://github.com/bbertucc/equalify/blob/main/CONTRIBUTING.md) guideline:

> Equalify's codebase only includes technology licensed with an [approved open-source license](https://opensource.org/licenses/category). Furthermore, our codebase only includes technology that can be freely loaded in every Equalify install. That means that core integrations cannot require API keys to function or installation procedures beyond Equalify's [official installation process](https://github.com/bbertucc/equalify#download-and-use).

Adoption of the policy would remove WAVE from Equalify's core codebase - I'll probably just add it in another repo. I would also need to integrate `axe-core` into Equalify's codebase to run accessibility scans "out-of-the-box." (Worth noting: @j-mendez created https://github.com/bbertucc/equalify/pull/153 to side-load A11yWatch. Merging A11yWatch would probably require an update of Equalify's [official installation process](https://github.com/bbertucc/equalify#download-and-use) to make them easy for any user to run A11yWatch with Equalify.

## Next Steps
Add any comments!

These updates to [CONTRIBUTING.md](https://github.com/bbertucc/equalify/blob/main/CONTRIBUTING.md) could be merged with 
[Beta-1](https://github.com/bbertucc/equalify/milestone/6).

**Comment by j-mendez** [comment]
The sidecar is more for development and can be optionally used in production ( not recommended for real work loads outside testing ).

When you deploy to production point the url to any valid external instance.

**Comment by j-mendez** [comment]
@bbertucc personal opinion just build what you think is correct and everything else falls in place. A lot of the highlights I am seeing are not appropriate to put on Github for the project. Keep the financial aspect away from the Open aspect of the software personally or else you will come to hate what you are building instead of building with a plan.

**Comment by bbertucc** [comment]
> When you deploy to production point the url to any valid external instance.

That's great. We'll need to refine these steps in #176. Adding some comments there.

**Comment by bbertucc** [comment]
Closing this with [6e52bc3](https://github.com/bbertucc/equalify/commit/6e52bc3674a356ad3f9b1fafd059f0193c1e45ba)

**Comment by bbertucc** [comment]
Merged into [beta-1](https://github.com/bbertucc/equalify/tree/beta-1).
